src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/TypeNameObject.cs

```diff
@@ -49,10 +59,14 @@ static TypeNameObject()
 
         int Add<T>(string? alias = null)
         {
-            var fullName = typeof(T).FullName!;
             Debug.Assert(knownTypeNames.Count < byte.MaxValue, "Too many known type names to fit in a byte index.");
+
+            var fullName = typeof(T).FullName!;
+            var @namespace = typeof(T).Namespace!;
```

Comment:
>Namespace! [](http://example.com/codeflow?start=39&length=10)

why this bang?

Reply:
just a left over from earlier changes. Will remove.